proxmox-ve.json

```diff
@@ -6,11 +6,13 @@
     "hyperv_switch_name": "{{env `HYPERV_SWITCH_NAME`}}",
     "hyperv_vlan_id": "{{env `HYPERV_VLAN_ID`}}",
     "apt_cache_host": "{{env `APT_CACHE_HOST`}}",
-    "apt_cache_port": "{{env `APT_CACHE_PORT`}}"
+    "apt_cache_port": "{{env `APT_CACHE_PORT`}}",
+    "output_base_dir": "{{env `PACKER_OUTPUT_BASE_DIR`}}"
   },
   "builders": [
     {
       "name": "proxmox-ve-amd64-libvirt",
+      "output_directory": "{{user `output_base_dir`}}/output-proxmox-ve-amd64-libvirt",
```

Comment:
can this use `{{ .Name }}`? as-in:

```json
"output_directory": "{{user `output_base_dir`}}/output-{{ .Name }}",
```

Reply:
Mhm, this creates directory `/dev/shm/output-<no value>`. Do you have any idea?

Reply:
oh, so that [template variable](https://www.packer.io/docs/templates/legacy_json_templates/engine) is not available there.

can you try with [build_name](https://www.packer.io/docs/templates/legacy_json_templates/engine#build_name)? as-in:

```json
"output_directory": "{{user `output_base_dir`}}/output-{{build_name}}",
```

if that does not work, never-mind, ignore my suggestion, and I'll close this comment.

Reply:
`{{build_name}}` works.